multiversx_sdk_core/transaction_builders/relayed_v2_builder_test.py

```diff
@@ -32,18 +32,19 @@ def test_without_arguments(self):
     def test_with_inner_tx_gas_limit(self):
         builder = RelayedTransactionV2Builder()
         network_config = NetworkConfig()
+        transaction_computer = TransactionComputer(AddressConverter())
 
         inner_tx = Transaction(
             chain_id=network_config.chain_id,
-            sender=Address.from_bech32(self.alice.label),
-            receiver=Address.from_bech32("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"),
+            sender=self.alice.label,
+            receiver="erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u",
             gas_limit=10,
             nonce=15,
-            data=TransactionPayload.from_str("getContractConfig")
+            data=TransactionPayload.from_str("getContractConfig").data
```

Comment:
Yup. I think `TransactionPayload` should be removed. It's not captured within the specs, and indeed, it should not be. We'll still keep it for some time in js-core, but can we perhaps remove it from py-core?

Reply:
It's now removed from the tests.